src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.html

```diff
@@ -145,10 +120,22 @@
 
 <ng-template #imageUsageTpl
              let-row="row">
-  <cd-usage-bar *ngIf="row"
-                [total]="row.size"
-                [used]="row.disk_usage"
-                [title]="row.name"
-                decimals="2">
+  <span *ngIf="!row.features_name.includes('fast-diff') || row.mirror_mode === 'snapshot' ; else usageBar"
+        [ngbTooltip]="usageTooltip">
+    <div>-</div>
```

Comment:
@nizamial09 @Pegonzal Can we use `N/A` or have it blank to be consistent?

Reply:
that would be contradictory to what we decided here: https://github.com/ceph/ceph/pull/51561#issuecomment-1557474940